pkg/network/node/networkpolicy.go

```diff
@@ -456,6 +456,13 @@ func (np *networkPolicyPlugin) parseNetworkPolicy(npns *npNamespace, policy *net
 					npp.watchesNamespaces = true
 					peerFlows = append(peerFlows, np.selectNamespaces(peer.NamespaceSelector)...)
 				}
+			} else if peer.IPBlock != nil {
+				// As per https://github.com/kubernetes/kubernetes/blob/master/pkg/apis/networking/types.go#L181
+				if peer.NamespaceSelector != nil || peer.PodSelector != nil {
+					return nil, fmt.Errorf("PodSelector or NamespaceSeletor can not be used along with IPBlocks")
+				}
+				// TODO: Add logic for "Except" block
+				peerFlows = append(peerFlows, fmt.Sprintf("Flow rule with CIDR =%s successfully added", peer.IPBlock.CIDR))
```

Comment:
obviously this needs to be changed to real openflow...

Reply:
@danwinship thanks for your reviews. I didnt quite understand your comment. Can you please explain :)

Reply:
peerFlows is an array of openflow expressions to match traffic against. That is where all the real work of implementing NetworkPolicy happens. You can't just *say* "Flow rule successfully added", you have to actually write the correct flow rule.